.github/workflows/tests.yml

```diff
@@ -144,3 +150,129 @@ jobs:
         run: |
           source venv/bin/activate && pytest test_downloads.py --base_url="$OUTPUT_CONTAINER_URL" --report_dir="$REPORT_DIR"
         working-directory: ./tests
+
+      - name: Prepare directory name for tests with object by object replacement
+        shell: bash
+        env:
+          TIMESTAMP: ${{ steps.date.outputs.timestamp }}
+        run: |
+          echo "REPL_DATA_DIR=${{ github.run_number }}-$TIMESTAMP-$(uuidgen)-obj-by-obj-replacement" >> $GITHUB_ENV
+
+      - name: Create a directory tree for tests with objects replacement
+        shell: bash
+        run: |
+          mkdir "$REPL_DATA_DIR"
+
+      - name: Move files to the directory tree for tests with url prefix
+        shell: bash
+        env:
+          SOURCE_DIR: ${{ env.TESTS_DATA_DIR }}
+          DEST_DIR: ${{ env.REPL_DATA_DIR }}
+        run: |
+          rsync -av "$SOURCE_DIR" "$DEST_DIR"
```

Comment:
Why rsync, btw? `cp -a`?

Reply:
I like rsync. Why not?

Reply:
It's handy for both operations, but I  replaced rsync to `cp -a`